src/Components/UI/Navigation/NavBarBottom.jsx

```diff
@@ -11,7 +11,7 @@ import { Link, useLocation } from "react-router-dom";
 export const NavBarBottom = () => {
   const { pathname } = useLocation();
 
-  const activeLeaderboard = pathname.includes("leadeboard");
+  const activeLeaderboard = pathname.includes("leaderboard");
   const activeCategory = pathname.includes("category");
```

Comment:
Since this are just strings chances of missing a character is there which is result in issue in the app. May be add constant variables and store the strings values there and use the variables.